frontend/src/metabase/parameters/utils/mapping-options.ts

```diff
@@ -42,15 +45,15 @@ function buildStructuredQuerySectionOptions(
   query: Lib.Query,
   stageIndex: number,
   group: Lib.ColumnGroup,
+  columns: Lib.ColumnMetadata[],
 ): StructuredQuerySectionOption[] {
   const groupInfo = Lib.displayInfo(query, stageIndex, group);
-  const columns = Lib.getColumnsFromColumnGroup(group);
 
   return columns.map(column => {
     const columnInfo = Lib.displayInfo(query, stageIndex, column);
 
     return {
-      sectionName: getColumnGroupName(groupInfo),
+      sectionName: getColumnGroupName(groupInfo) || t`Summaries`,
```

Comment:
Can we make this something more unique? I'm getting duplicate names now for [certain (convoluted) questions](http://localhost:3000/question#eyJkYXRhc2V0X3F1ZXJ5Ijp7ImRhdGFiYXNlIjoxLCJ0eXBlIjoicXVlcnkiLCJxdWVyeSI6eyJleHByZXNzaW9ucyI6eyJEb2JUYXgiOlsiKiIsWyJmaWVsZCIsIm1pbiIseyJiYXNlLXR5cGUiOiJ0eXBlL0Zsb2F0In1dLDJdfSwiYWdncmVnYXRpb24iOltbIm1heCIsWyJleHByZXNzaW9uIiwiRG9iVGF4Iix7ImJhc2UtdHlwZSI6InR5cGUvRmxvYXQifV1dXSwiYnJlYWtvdXQiOltbImZpZWxkIiwiRElTQ09VTlQiLHsiYmFzZS10eXBlIjoidHlwZS9GbG9hdCIsImJpbm5pbmciOnsic3RyYXRlZ3kiOiJkZWZhdWx0In19XV0sInNvdXJjZS1xdWVyeSI6eyJzb3VyY2UtdGFibGUiOjUsImV4cHJlc3Npb25zIjp7IkRvdWJsZSBEaXNjb3VudCI6WyIqIixbImZpZWxkIiwzNix7ImJhc2UtdHlwZSI6InR5cGUvRmxvYXQifV0sMl19LCJhZ2dyZWdhdGlvbiI6W1sibWluIixbImZpZWxkIiwzOCx7ImJhc2UtdHlwZSI6InR5cGUvRmxvYXQifV1dXSwiYnJlYWtvdXQiOltbImZpZWxkIiwzNix7ImJhc2UtdHlwZSI6InR5cGUvRmxvYXQiLCJiaW5uaW5nIjp7InN0cmF0ZWd5IjoiZGVmYXVsdCJ9fV1dfX19LCJkaXNwbGF5IjoidGFibGUiLCJkaXNwbGF5SXNMb2NrZWQiOnRydWUsInBhcmFtZXRlcnMiOltdLCJ2aXN1YWxpemF0aW9uX3NldHRpbmdzIjp7InRhYmxlLmNlbGxfY29sdW1uIjoiYXZnIn0sIm9yaWdpbmFsX2NhcmRfaWQiOjI2MCwidHlwZSI6InF1ZXN0aW9uIn0=).

<img width="1141" alt="Screenshot 2024-08-16 at 15 51 42" src="https://github.com/user-attachments/assets/6b746226-b1b3-4345-baae-eba31508aa6a">

Reply:
1. The filter modal has not been affected by this PR, it's an existing thing.
2. Yes! @romeovs can you please report it as a bug? It'll be similar to #46845

Reply:
Yes, let's not improve it here

Reply:
> can you please report it as a bug? 

Reported separately as it is an existing issue: #47001